CMake/Servus.cmake

```diff
@@ -1,6 +1,16 @@
 
 set(SERVUS_PACKAGE_VERSION 1.0)
-set(SERVUS_REPO_URL https://github.com/HBPVis/servus.git)
-set(SERVUS_DEPENDS OPTIONAL DNSSD avahi-client Boost)
+set(SERVUS_REPO_URL https://github.com/HBPVis/Servus.git)
+set(SERVUS_SOURCE "${CMAKE_CURRENT_SOURCE_DIR}/src/Servus")
+set(SERVUS_DEPENDS hbpvis REQUIRED Boost Threads OPTIONAL DNSSD avahi-client)
```

Comment:
Probably. We only suffer through all this as we don't want Boost anymore being required, so that's why I was puzzled...